vscode/README.md

```diff
@@ -59,6 +59,12 @@ In the VS Code command palette :
 Project Explorer provides an overview of logical project structure, groups sources together and greatly simplifies Java package structure exploration. Project Explorer is an addition to the classical workspace explorer. Use it to build, test, execute and operate your Maven and Gradle Java projects.  
 ![Project Explorer](images/project-explorer.png)
 
+### Localisation support
+Oracle Java Platform extension provides localisation support for _Japanese_ and _Simplified Chinese_.
+To switch language `Go to command Palette->Configure display language` select `ja` for _Japanese_ or `zh-cn` for _Simplified Chinese_
+![localisation.gif](./images/localisation.gif)
```

Comment:
nitpick: For the sake of uniformity can we replace `./images/localisation.gif` with `images/localisation.gif`